clients/admin-ui/src/features/connection-type/connection-type.slice.ts

```diff
@@ -60,6 +62,13 @@ export const connectionTypeSlice = createSlice({
       ...state,
       connectionOption: action.payload,
     }),
+    setConnectionOptions: (
```

Comment:
With both this and `setConnection`, it looks like Ops code has the [same pattern of duplicating RTK query results](https://github.com/ethyca/fides/pull/1083#Description%20Of%20Changes:~:text=Query%20vs%20slices.-,TLDR,-%3A%20Use%20useFooQuery%20and) that Ctl did. Check out what I wrote there.

For this code, instead of storing & selecting this state, we can use some code like:
```ts
const selectConnectionTypes =  createSelector(
    connectionTypeApi.endpoints.getAllConnectionTypes.select(),
    ({ data }) => data?.items ?? []
  );
```

Reply:
Issue has been resolved now